services/workflows-service/src/alert/alert.repository.ts

```diff
@@ -42,6 +42,33 @@ export class AlertRepository {
     return await this.prisma.alert.findFirstOrThrow(queryArgs);
   }
 
+  // Method to update an alerts
+  async updateMany<T extends Omit<Prisma.AlertUpdateManyArgs, 'where'>>(
+    alertIds: string[],
+    projectId: TProjectId,
+    args: Prisma.SelectSubset<T, Omit<Prisma.AlertUpdateManyArgs, 'where'>>,
+  ): Promise<Alert[]> {
+    if (!projectId)
+      throw new Error('Project ID is required to perform an update opeartion on Alerts');
```

Comment:
Should it be `Error` and not one of our custom classes?
```suggestion
    if (!projectId) {
      throw new Error('Project ID is required to perform an update opeartion on Alerts');
   };
```